src/quo2/reanimated.cljs

```diff
@@ -47,7 +47,7 @@
        (into {})))
 
 ;; Worklets
-(def worklet-factory (js/require "../js/worklet_factory.js"))
+(def worklet-factory (js/require "../src/js/worklet_factory.js"))
```

Comment:
./js/worklet_factory.js ?

Reply:
For some reason `js/require` is only working with `../src/js/worklet_factory.js`


https://user-images.githubusercontent.com/17097240/175988135-909b2ee2-a4fe-4ad5-ae84-1aaf30bdc953.mp4